src/components/Item/Item.jsx

```diff
@@ -0,0 +1,56 @@
+import defaultItemImg from "../../assets/default/default_item.png";
+import {
+  ItemImgStyle,
+  ItemNameStyle,
+  ItemPriceStyle,
+  FavoriteContainer,
+  FavoriteCountStyle,
+  FavoriteImageStyle,
+  ItemUlStyle,
+  ItemIlStyle,
+} from "./Item.styles";
+
+function ItemList({ item, variant }) {
+  const { name, price, images, favoriteCount } = item;
```

Comment:
💊 제안
ItemList 라는 객체로 props를 받기보다 images, name... 이렇게 받으시는게 더 좋아보여요.
지금 해당 컴포넌트에서 variant 와 item이 동일한 위계로 보여서 객체로 받으실 필요가 없어보입니다~

```suggestion
function ItemList({ name, price, images, favoriteCount, variant }) {
```

Reply:
❗️ 수정요청
boolean 값임을 알 수 있게 variant가 아닌 다른 이름이면 좋겠습니다~